README.md

```diff
@@ -74,14 +76,16 @@ const vpPayload: PresentationPayload = {
 
 const vpJwt = await createPresentation(vpPayload, issuer)
 console.log(vpJwt)
```

Comment:
@mirceanis I think you forgot to update this example

Reply:
good catch!
I updated the readme to fix the _create_ examples.
Please take another look